testing-framework/README.md

```diff
@@ -128,4 +128,31 @@ Be aware that state files can contain sensitive information. Do not expose it to
 
 ## Installation via Github Actions
 
-(coming soon)
+To deploy a ephemeral devnet with Github CI/CD the workflow `.github/workflows/ephemeral_devnet_aws_deploy.yml` can be triggered for AWS and `.github/workflows/ephemeral_devnet_aws_deploy.yml` to trigger the workflow for hetzner deployment.
```

Comment:
`.github/workflows/ephemeral_devnet_aws_deploy.yml` twice here, I suspect second one should be different

Reply:
correct, this is updated now.